client/blocks/reader-featured-video/index.jsx

```diff
@@ -62,27 +75,29 @@ class ReaderFeaturedVideo extends React.Component {
 	}
 
 	componentDidMount() {
-		window.addEventListener( 'resize', throttle( this.updateVideoSize, 100 ) );
+		if ( this.props.allowPlaying ) {
+			window.addEventListener( 'resize', throttle( this.updateVideoSize, 100 ) );
+		}
```

Comment:
Apparently not! I see we've fixed that in https://github.com/Automattic/wp-calypso/pull/11743. I'll rebase this PR when that lands.